content/2021/pages/22.md

```diff
@@ -0,0 +1,84 @@
+---
+section: Politechnika w liczbach
+bgcImage: /images
+pageNumber: 22
+---
+
+Rekrutacja na studia  
+na Politechnice Białostockiej  
+odbywa się drogą elektroniczną  
+przez system Internetowej  
+Rejestracji Kandydatów (IRK):  
+[irk.pb.edu.pl](https://irk.pb.edu.pl)
+
+Założone w nim konto  
+kandydata służy nie tylko  
+do zapisów na studia,  
+ale też do przekazywania  
+wyników kolejnych etapów  
+postępowania rekrutacyjnego  
+i innych przydatnych  
+informacji.
+
+O zakwalifikowaniu na studia  
+decyduje pozycja na liście  
+rankingowej kandydatów  
+objętych postępowaniem  
+kwalifikacyjnym na poszczególnych  
+kierunkach studiów  
+(w ramach liczby oferowanych  
+miejsc).
+
+Kwalifikacja na studia  
+pierwszego stopnia przebiega  
+następująco:
+
+## 1. Kandydat umieszczany jest
+
+na listach rankingowych wszystkich  
+wybranych kierunków.  
+Na każdym kierunku ustala się  
+grupę osób zakwalifikowanych  
+na studia (zgodnie z liczbą  
+oferowanych miejsc).
+
+## 2. Jeżeli kandydat znajdzie się
+
+w grupie osób zakwalifikowanych  
+na więcej niż jeden kierunek,  
+pozostawia się jego nazwisko  
+tylko na kierunku najwyższego  
+priorytetu i wykreśla się go  
+z kierunku/kierunków  
+o priorytetach niższych.
+
+## 3. Wyniki kwalifikacji
+
+podawane są do publicznej  
```

Comment:
dalej nie zrozumiane. To powinno iść jednym ciągiem a nie jak w grafice + trzymajmy się tego że to język polski więc zdanie zaczyna się wielką literą i nie ma entera co 3 słowa